### PR TITLE
updated configure_base for integrating ArchiverEventSources

### PR DESCRIPTION
**Pull request by malitsky**
updated configure_base for integrating ArchiverEventSources

### CODE REVIEW

File: nslsii/__init__.py [hunk 1]
```diff
@@ -179,6 +179,11 @@ def configure_base(user_ns, broker_name, *,
     import bluesky.simulators
     import_star(bluesky.simulators, ns)
 
+    from databroker.eventsource.archiver import *
```
Comment (tacaswell) [inline nslsii/__init__.py]: Please do not use `import *`

**Comment by tacaswell** [comment]
I think we want to merge the base configuration into `db.named` instead of having a separate lookup system.

Do we want to allow `configure_base` to make additional PVs to be included?  That seems like something that will be in flux for a while ?   For now I guess just walking the `db.EventSources` and checking the type is the easiest way to get the right EventSource?  I suspect we want to also include some notion of 'name' in the config when creating the EVentSource objects and have some access to that from the DataBroker, ex `db.ES['archiver']` or `db.ES['raw']` ?

**Comment by danielballan** [comment]
:+1: for putting the list of PVs into the databroker configuration file.

**Comment by malitsky** [comment]
integration is implemented with the Broker API using the databroker configuration file and db.ES  as demonstrated by the following script and configuration file:

https://github.com/malitsky/nsls2/blob/master/areventsource/notebooks/archiver-reader.db.5.13.18.ipynb
https://github.com/malitsky/nsls2/blob/master/areventsource/notebooks/csx_with_archivers.yml

**Pull request closed without merging**